.travis.yml

```diff
@@ -10,7 +10,8 @@ before_install:
   - npm install -g grunt@0.4.5
   - npm install -g grunt-cli
 install: npm install
-scripts:
+script:
+  - grunt
```

Comment:
Yes, the idea is that `npm test` would invoke `grunt`, but the bin would come from the node_modules folder instead of requiring a global install.